frontend/public/components/utils/datetime.ts

```diff
@@ -3,8 +3,32 @@ import * as moment from 'moment';
 
 const getLocale = () => localStorage.getItem('bridge/language');
 
-export const getLocaleDate = (date: Date, options): string =>
-  date.toLocaleDateString(getLocale() || undefined, options);
+export const timeFormatter = new Intl.DateTimeFormat(getLocale() || undefined, {
```

Comment:
Note that this will not be reactive to language preference changes. I'm debating whether we should just always use `undefined` here and use the system locale. Date formatting seems like it should be a separate preference, particularly since we can support any locale, not just the ones we have translations for.

Reply:
It won't be reactive without refreshing, but it will still be active.  I'm thinking it's better than not having it at all.

Reply:
We can make it reactive, but we'd need a `useEffect` hook with the locale as a dependency everywhere we use it. Maybe it's worth it.

Or we could create a new formatter each render.

Reply:
A new formatter each render wouldn't perform as well when using `Timestamp` in lists, though, since we'd be creating a formatter for every row.